.travis.yml

```diff
@@ -75,7 +90,7 @@ script:
     cp $TRAVIS_BUILD_DIR/opencl.jam ~/
     cp $TRAVIS_BUILD_DIR/clblas.jam ~/
   - cd libs/numeric/ublas
-  - $BOOST_ROOT/b2 -j 8 test toolset=$TOOLSET cxxstd=$CXXSTD
+  - travis_wait 120 sleep infinity & $BOOST_ROOT/b2 -j 8 test/tensor toolset=$TOOLSET cxxstd=$CXXSTD
```

Comment:
@bassoy I think you missed this while review.

Did you intensionally disable the testing to `non-tensor` based tests?

@amitsingh19975 For Testing on our GSOC fork and speed, We had disabled unit tests of complete uBLAS and only kept tensor tests.

There is no action required from your side when I propose the new CI, I will re-enable the complete unit tests.

Reply:
@coder3101 
Thanks for pointing this out. Yes I have missed it when reviewing.

Reply:
:+1: